### PR TITLE
[EntityListenerPass] Handle ChildDefinition

### PR DESCRIPTION
**Pull request by fancyweb**
Fixes https://github.com/doctrine/DoctrineBundle/pull/989#discussion_r309244615

### CODE REVIEW

**Comment by alcaeus** [comment]
Ping @doctrine/team-symfony-integration - I'm not very familiar with child definitions and the changes necessary to support them, so I'd appreciate some input. Thanks!

File: DependencyInjection/Compiler/EntityListenerPass.php [hunk 1]
```diff
@@ -87,40 +88,57 @@ public function process(ContainerBuilder $container)
...
+            $resolverClass = $container->getParameterBag()->resolveValue($resolverClass);
         }
 
         return $resolverClass;
     }
+
+    private function getConcreteDefinitionClass(Definition $definition, ContainerBuilder $container, string $id) : string
+    {
+        $class = $definition->getClass();
+        if (is_string($class)) {
```
Comment (nicolas-grekas) [inline DependencyInjection/Compiler/EntityListenerPass.php]: we don't usually check with `is_string` in similar situations, it should be just `if ($class) {` to me

Comment (SenseException) [reply DependencyInjection/Compiler/EntityListenerPass.php]: @fancyweb Do you have this change request on your todo list?

Comment (fancyweb) [reply DependencyInjection/Compiler/EntityListenerPass.php]: Just done it.

**Comment by alcaeus** [comment]
Thanks @fancyweb!

**Pull request merged**